sklearn/datasets/base.py

```diff
@@ -255,6 +255,12 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object
+
     Returns
     -------
     data : Bunch
```

Comment:
Well on line 266 in the diff it still says that it returns `data : Bunch`. Would it be apt to include information about it possibly returning (data, target) as well?